pages/installation.md

```diff
@@ -142,9 +142,9 @@ for Eglot:
 
 Click on the extensions button on the sidebar, then search for
 `lexical`, then click `install`.  By default, the extension will automatically
-download the latest version of Lexical.
+download the latest version of Expert.
 
-To change to a local executable, go to `Settings -> Extensions -> Lexical` and
+To change to a local executable, go to `Settings -> Extensions -> Expert` and
```

Comment:
I believe these ones are supposed to be Lexical still since the Lexical extension is being used.

Reply:
I've noticed that getting it running locally. In neovim, if I set `lspconfig.expert.setup { ...` it wouldn't work, but `lspconfig.lexical.setup` did. I'm happy to help out with any other renaming stuff

Reply:
For the VSCode extension, we should keep Lexical here since that's the extension that is compatible with Expert at the moment, we need a new Expert vscode extension

On Neovim this is my configuration: https://github.com/doorgan/dotfiles/blob/main/dot_config/nvim/lua/plugins/lsp-config.lua#L26-L55 we should update the docs to be more inline with this

Reply:
From the `nvim-lspconfigs` github, it appears that `require('lspconfig')` is deprecated now? https://github.com/andyleclair/nvim-lspconfig?tab=readme-ov-file#important-%EF%B8%8F

IMO, if that's what _they're_ saying, it's what _we_ should say?

Reply:
Ok, I have this configured in my config, confirmed working in neovim `0.11.3`

https://github.com/andyleclair/.dotfiles/blob/main/.config/nvim/lua/custom/plugins/elixir.lua

<img width="1586" height="686" alt="image" src="https://github.com/user-attachments/assets/2fdf911b-f865-4ba1-8c83-fc7b2901ac38" />

If you like, I can update the docs for that? I've got a PR cooking for `nvim-lspconfig` that essentially has the same stuff, I just wanted to make sure it worked first!

Reply:
Awesome, let's do it, thank you! 💜